### ISSUE TITLE
Logo or any other png image doesnot work with transparent background

### ISSUE THREAD

**Issue by SATYADAHAL** [emacs-dashboard/emacs-dashboard#458]
![ScreenShot](https://user-images.githubusercontent.com/51309409/236814705-ef8dd0c3-dfe2-420b-8d4b-2a54abf7e24f.png)

When I try to enable transparent backgrounds in Emacs using the "alpha-background" function, it unfortunately does not result in a transparent background for PNG images. Instead, it applies the current theme's background color to the image.

**Comment by jcs090218** [comment]
Not every OS supported this; that's why I haven't looked into this. Duplicate of https://github.com/emacs-dashboard/emacs-dashboard/discussions/376.

This is indeed a good feature we would like to support!

**Comment by SATYADAHAL** [comment]
I attempted to insert a large ASCII art, but its size was equivalent to that of other faces, causing it to take up all the available space. To address this, I attempted to scale down the text in the dashboard buffer by -10 and adjust all other faces to their normal size. However, as the centering of the ASCII art was not taken into account during the text scaling, the art was no longer centered. It would be beneficial if the centering of the ASCII art could be considered while scaling the text, as well.
By creating an ASCII art that is equivalent in size to a PNG image, it could potentially serve as a substitute for the image. However, the loss of color would be a trade-off.
But it would come with a lots of bugs since all the faces are resized to smaller version.Here is my code that i tried to get
the result below.
```
(add-hook 'dashboard-mode-hook
(lambda () (text-scale-set -10)))

(custom-set-faces
  '(dashboard-banner-logo-title ((t (:height 140 :weight bold :inherit))))
  '(dashboard-heading ((t (:height 140 :weight bold :inherit))))
  '(dashboard-items-face ((t (:height 140 :inherit))))
  )
```
![Screenshot_20230509_104543](https://user-images.githubusercontent.com/51309409/236999095-e85cec47-fff7-4818-b02f-6bb5f9acc3fe.png)

**Comment by Schievel1** [comment]
That does not work for SVGs for you either, does it? I am using SVG as the logo and it does the same. I am wondering if I am hunting the same problem like you do here.

**Comment by SATYADAHAL** [comment]
It's the same with SVG images too. Emacs simply replaces the transparent parts of the image with the theme color, effectively removing the transparency.

**Comment by SATYADAHAL** [comment]
> But I only have this since emacs 29. On 28 it worked most of the times.

Which OS are you using?

**Comment by Schievel1** [comment]
Gentoo Linux

**Comment by SATYADAHAL** [comment]
After googling and doing some research i found out that,until emacs doesnot change the way it displays transparent images. I don't see a way to solve this issue.

**Comment by Icy-Thought** [comment]
I was also told that it was an Emacs issue rather than an emacs-dashboard issue in the Emacs matrix room. Might want to submit an issue in the main project repository.

**Comment by o0nd7ots** [comment]
Please see #481. Yep. it is an emacs issue. I suggest you use `.webp` and close this issue.

**Comment by o0nd7ots** [comment]
However respect for coming up with the ASCII workaround.
I was thinking about adding it as a feature but I think it would be redundant.

**Comment by SATYADAHAL** [comment]
I made it work with xpm images, but it had some issues, such as losing the smoothness around the image. However, it's a better option compared to using PNG, as it removes the background. I downloaded the PNG image and converted it to XPM using this [site](https://convertio.co/png-xpm/). Then, I added the image path to the configuration as follows:
`(setq dashboard-startup-banner "~/.emacs.d/dashboardBanners/omBlue.xpm")`
Now it looks like this.
![Screenshot_20231030_161359](https://github.com/emacs-dashboard/emacs-dashboard/assets/51309409/322a0a77-a59e-4a2e-84ed-316e9a87088c)

Since this is not emacs-dashboard issue. I will be closing it.
